Source/TestDafny/MultiBackendTest.cs

```diff
@@ -354,7 +355,7 @@ int ExpectedExitCode
     }
 
     // If we hit errors, check for known unsupported features or bugs for this compilation target
-    if (error == "" && OnlyAllowedOutputLines(backend, outputString)) {
+    if (error == "" && HasUnsupported(backend, outputString)) {
```

Comment:
Doesn't this do the wrong thing if there are unsupported feature lines but also some unrelated crash?

Reply:
I'll see if I can change it back. Might have been for something that changed already.